ddtrace/debugging/_debugger.py

```diff
@@ -229,11 +229,11 @@ def _open_contexts(self) -> None:
             contexts.append(self._collector.attach(signal))
 
         # Save state on the wrapping context
-        self.set("start_time", compat.monotonic_ns())
+        self.set("start_time", time.monotonic_ns())
```

Comment:
Would we save much if we imported `monotonic_ns` and called it directly, instead of every time looking it up from the `time` module?

Reply:
Is this something we should do just here, or on every place using `monotonic_ns`, or on every place using any of these functions?

Reply:
If it's not much hassle I think we can do it here. WDYT?

Reply:
Done!